### ISSUE TITLE
CarrierAccount.types api_key not handled correctly

### ISSUE THREAD

**Issue by eprussell** [EasyPost/easypost-python#67]
Seems like when there's a field `api_key` in a carrier type this isn't behaving as intended with this client library. 

Example below is Australia Post but this appears to happen for any carrier with api_key among the fields (e.g. Parcel, Fastway, etc.)

result with cURL:
```
curl -X GET https://api.easypost.com/v2/carrier_types   -u PRODUCTION_API_KEY:
```
```
{
 "object": "CarrierType",
 "type": "AustraliaPostAccount",
 "readable": "Australia Post",
 "logo": null,
 "fields": {
  "credentials": {
   "account_number": {
    "visibility": "visible",
    "label": "Australia Post Account Number"
   },
   "api_key": {
    "visibility": "masked",
    "label": "Australia Post API Key"
   },
   "api_secret": {
    "visibility": "password",
    "label": "Australia Post Secret Key"
   },
   "print_as_you_go": {
    "visibility": "checkbox",
    "label": "Print as you go"
   }
  }
 }
},
```
result with python client library:
```
import easypost
easypost.api_key = 'api_key_here'
carrier_types = easypost.CarrierAccount.types()
print carrier_types
```
```
<EasyPostObject CarrierType at 0x11007abd0> JSON: {
  "fields": {
    "credentials": {
      "account_number": {
        "label": "Australia Post Account Number",
        "visibility": "visible"
      },
      "api_secret": {
        "label": "Australia Post Secret Key",
        "visibility": "password"
      },
      "print_as_you_go": {
        "label": "Print as you go",
        "visibility": "checkbox"
      }
    }
  },
  "logo": null,
  "object": "CarrierType",
  "readable": "Australia Post",
  "type": "AustraliaPostAccount"
},
```

Note in the Python case the absence of `api_key`

**Comment by Roguelazer** [comment]
that seems entirely plausible

**Comment by Roguelazer** [comment]
This is fixed in version 3.6.3, now available [in pypi](https://pypi.python.org/pypi/easypost).
